companies/forms.py

```diff
@@ -359,6 +359,15 @@ class CompanySearchForm(forms.Form):
 		('YES', 'Show only exhibitors')
 	]
 
+	year_choices = [
```

Comment:
It would be great if this could be derived from the current year, so that it doesn't need to be updated for each coming year.